packages/docs/src/pages/guidelines/accessibility.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
```

Comment:
oops I thought I posted this already, but i guess not

The guiding principles page only mentions accessibility once, and it feels like a really outdated guide. It has very specific info on class naming and principles behind css utilities, but nothing on a more general product principles like accessibility. I wonder if we should remove this bullet for now and work on writing a new guiding principles to include in our doc site or in the GH wiki

Reply:
I think that works